.github/workflows/deploy.yml

```diff
@@ -87,9 +86,24 @@ jobs:
       service-account: ${{ secrets.DEV_STANDALONE_TOKEN }}
       registry-pass: ${{ secrets.HARBOR_REGISTRY_PASSWORD }}
 
+  deploy-dev-cvdmirror:
+    name: dev cvdmirror
+    needs: conditions
+    if: needs.conditions.outputs.dev-cvdmirror == 'true'
+    uses: bratislava/github-actions/.github/workflows/deploy-with-bratiska-cli-inhouse.yml@beta
```

Comment:
I'm not totally sure if this should be `@beta` couse of DEV or it needs to be changed before release to `@stable`.

Reply:
I usually leave `dev` deployments with the `beta` tag, which I use for testing future updates on pipelines. It’s good to keep it distinct from `stable`.